docs/BindingsGuide_Java.md

```diff
@@ -1,10 +1,39 @@
 # Azure SQL bindings for Azure Functions - Java
 
+## Setup Function App
+1. Install [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local)
+
+2. Create a function app for Java:
+    ```bash
+    mkdir MyApp
+    cd MyApp
+    func init --worker-runtime java
+    ```
+
+3. Enable SQL bindings on the function app. More information can be found [in Microsoft Docs](https://docs.microsoft.com/azure/azure-functions/functions-bindings-azure-sql).
+
+    Update the `host.json` file to the preview extension bundle.
+    ```json
+    "extensionBundle": {
+        "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+        "version": "[4.*, 5.0.0)"
+    }
+    ```
+
 ## Input Binding
 
 ### SQLInput Attribute
 
-_TODO_
+In the Java functions runtime library, use the @SQLInput annotation (com.microsoft.azure.functions.sql.annotation.SQLInput) on parameters whose value would come from Azure SQL. This annotation supports the following elements:
+
+| Element |Description|
+|---------|---------|
+| **commandText** | Required. The Transact-SQL query command or name of the stored procedure executed by the binding.  |
+| **connectionStringSetting** | The name of an app setting that contains the connection string for the database against which the query or stored procedure is being executed. This value isn't the actual connection string and must instead resolve to an environment variable name. | 
```

Comment:
This is different than the other connection string setting descriptions you have, let's make sure these are all the same

(note the other ones - including your powershell one - have broken links for the extra connection string settings. So make sure to fix that too)